UPGRADING.md

```diff
@@ -298,3 +298,9 @@ const MyComponent = () => {
 - The `react-query` query key for the `useMe()` data changed. The hook is now
   powered by `useRestQuery`. If needing to interact with the cached data, use
   the `useRestCache` hook.
+
+### 11.x -> 12.x
```

Comment:
I don't think this is too big of a deal, but FWIW it looks like this this PR didn't result in a major semantic release.  e.g. Latest is 11.9.1.  Just documenting here so we can fix this up next time we're in UPGRADING